### DISCUSSION TITLE
How to work with embedding?

### DISCUSSION THREAD

**Discussion by Kapysta38** [oobabooga/text-generation-webui#6242] [Q&A]
I don't understand how to connect embeddings to my model, how to use them correctly when interacting via API (extension openai). I want to use other embedding, and how to change them I found [here](https://github.com/oobabooga/text-generation-webui/wiki/12-%E2%80%90-OpenAI-API). But unfortunately I have not found a detailed description of how they work. Could you please give me some links or help me to understand it?

**Comment by aallsbury** [comment]
+1 This!
I have been trying to figure this out for a while. Hoping there is something similar to the tabby API situation where you can use cuda or cpu to host the embedding model and it populates under the OpenAI api endoint for embedding.

**Reply by Kapysta38** [reply]
I was able to achieve some results, but I don't have time to share them right now. I'll get back to you later.

**Reply by aallsbury** [reply]
Great to hear! please let me know when you can!
